platform/servers_list.cpp

```diff
@@ -11,32 +11,53 @@
 namespace downloader
 {
 // Returns false if can't parse urls. Note that it also clears outUrls.
-bool ParseServerList(std::string const & jsonStr, std::vector<std::string> & outUrls)
+bool ParseMetaConfig(std::string const & jsonStr, MetaConfig & outMetaConfig)
 {
-  outUrls.clear();
+  outMetaConfig = MetaConfig();
   try
   {
     base::Json root(jsonStr.c_str());
-    for (size_t i = 0; i < json_array_size(root.get()); ++i)
+    json_t *servers;
+    if (json_is_object(root.get()))
     {
-      char const * url = json_string_value(json_array_get(root.get(), i));
+      // New format:
+      // {
+      //   "servers": ["http://url1", "http://url2", "http://url3"],
+      //   "settings": {
+      //      "key1": "value1",
+      //      "key2": "http://url"
+      //    }
+      // }
+
+      json_t *settings = json_object_get(root.get(), "settings");
```

Comment:
Not our code style here and in other places. Also const?

Reply:
Sorry, I will change to `json_t* settings`. 

> Also const?

Underlying code returns json_t without const.

```
json_t *json_object_get(const json_t *json, const char *key)
```

Reply:
If you don't intend to modify it, you should always mark it as const. See our and Google's C++ code style guidelines. Let's not create exceptions and follow the code style everywhere.

Reply:
I added consts where they can be added. Unfortunally, but

```
void *json_object_iter(json_t *object);
```